components/time/src/zone/mod.rs

```diff
@@ -203,19 +224,29 @@ pub(crate) mod ule {
     #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
     #[non_exhaustive]
     pub enum TimeZoneVariant {
-        /// The variant corresponding to `"standard"` in CLDR.
+        /// Indicates the period in a time zone with a lower UTC offset than `Daylight`.
```

Comment:
I wouldn't define these as "periods"

```suggestion
        /// Indicates the variant of a time zone with a lower UTC offset than the `Daylight` variant.
```

Reply:
Changing "variant of" to "period in" is one of the main changes in this PR. I think "period in" more accurately reflects that we're talking about a time zone reckoning for only part of the year.